service/ziti-tunnel/service/pkg-vars.go

```diff
@@ -75,6 +75,11 @@ const (
 	ShutdownEvent    = 0 //7
 	ErrorEvent       = 0 //1000
 
+	PBT_APMSUSPEND = 4 // Entering Suspend
```

Comment:
might be good to link to the ms doc you found that spelled these out in here?

Reply:
I have added the ms doc as comment in the file